grails-app/views/shared/_projectFinderQueryPanel.gsp

```diff
@@ -16,7 +16,7 @@
         <g:render template="/shared/facetView" model="[modalName: 'chooseMore']"></g:render>
 
         <div class="filters-footer">
-            <button class="btn btn-sm btn-dark" data-target="#mapModal" data-toggle="modal">
+            <button class="btn btn-lg btn-dark" data-target="#mapModal" data-toggle="modal" style="font-size: .9rem;">
```

Comment:
General best practise is to add a class and not to style the element directly.

Reply:
Hi @temi what css file can I insert the class for this?

Reply:
project-finder.css